### PR TITLE
Fix layout glitch in safari [ED-621]

### PR DESCRIPTION
**Pull request by joshmarom**

### CODE REVIEW

File: assets/dev/js/frontend/frontend.js [hunk 1]
```diff
@@ -166,6 +166,14 @@ export default class Frontend extends elementorModules.ViewModule {
 		this.elements.$wpAdminBar = this.elements.$document.find( this.getSettings( 'selectors.adminBar' ) );
 	}
 
+	addUAClasses() {
```
Comment (matipojo) [inline assets/dev/js/frontend/frontend.js:169]:
```suggestion
	addUserAgentClasses() {
```

File: core/common/assets/js/utils/environment.js [hunk 2]
```diff
@@ -1,14 +1,49 @@
-const userAgent = navigator.userAgent,
+const hasUA = ( UAstr ) => {
+		return userAgent.indexOf( UAstr ) >= 0;
+	},
```
Comment (matipojo) [inline core/common/assets/js/utils/environment.js:3]: Please use the full term 'UserAgent' where it's possible.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/common/assets/js/utils/environment.js [hunk 3]
```diff
@@ -1,14 +1,49 @@
...
+		edge: isEdge,
+		firefox: isFirefox,
+		ie: isIE,
+		mac: matchUserAgent( 'Macintosh' ),
+		opera: isOpera,
+		safari: isSafari,
+		webkit: matchUserAgent( 'AppleWebKit' ),
 	};
 
-environment.chrome = -1 !== userAgent.indexOf( 'Chrome' ) && ! environment.opera && ! environment.edge;
```
Comment (matipojo) [inline core/common/assets/js/utils/environment.js:12]: Why are the additional checks removed?
 ` ! environment.opera && ! environment.edge;`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/common/assets/js/utils/environment.js (hunk 3 above)
Comment (joshmarom) [reply core/common/assets/js/utils/environment.js:12]: Good point.
We need to discuss this.

**Review by matipojo — DISMISSED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**